README.md

```diff
@@ -53,6 +53,33 @@ print('prediction: {}'.format(prediction.generations[0].text))
 
 There is also an asyncio compatible client called `cohere.AsyncClient` with an equivalent interface. Consult the [SDK Docs](https://cohere-sdk.readthedocs.io/en/latest/) for more details.
 
+If you are looking to work with finetunes you will have to instantiate a FinetuneClient
+```python
+import cohere
+from cohere.finetune_dataset import  InMemoryDataset, JsonlDataset
+
+co = cohere.Client('YOUR_API_KEY')
```

Comment:
maybe move down to where you use it as in 
# you can now call your model as usual
co = ..
co.generate